src/entity/Web.java

```diff
@@ -4,7 +4,7 @@
 
 import java.awt.*;
 
-public class Web extends Entity{
+public class  Web extends Entity{
```

Comment:
Why this line changed? I think it's unnecessary changes.

Reply:
A typo has been made. We'll change it back the next time we merge! 👍